CHANGELOG.md

```diff
@@ -6,6 +6,93 @@
 
 ## 4.0
 
+Below are the major highlights in JupyterLab 4.
+
+### New text editor
+
+CodeMirror, the text editor used for cells and file editors, has been updated to [CodeMirror 6](https://codemirror.net/). This brings important
+performance improvements as well as better customization capabilities.
+The editor settings have also been improved. Previously settings had to be individually set for each type of cell, the file editor, and the console editor. Now they are all set in one place, which makes it easier to use the default settings for all editors or change some settings for specific cases. For example, it is possible to hide the line numbers for markdown cells.
+
+Developers can now provide editor extensions, like themes and programming language parsers, through new application registries.
+
+### New extension manager
+
+Starting with JupyterLab 3, extensions can be installed via Python packages.
+
+In JupyterLab 4, building on this feature, the Extension Manager can now list and install extensions from [pypi.org](https://pypi.org/search/?c=Framework+%3A%3A+Jupyter+%3A%3A+JupyterLab).
+
+Developers can provide an alternative package repository to display their own set of extensions.
+
+### Improved document search
+
+The Search and Replace functionality has been improved with new features when searching in a notebook:
+
+- Highlight matches in rendered markdown cells
+- Search in selection
+- Multi-line search
+- Replace using regex capture-group references
+- Replace while preserving case
+
+### UI improvements
+
+Some new elements have been added or changed in the UI:
+
+- Rework the running kernels section
+- "Add cell" button at the notebook bottom
+- Dialog to display keyboard shortcuts as in the Classic Notebook (use <kbd>CTRL+SHIFT+H</kbd>)
+- Display first line of cell input and outputs when they are collapsed
+
+### Accessibility improvements
+
+JupyterLab is not yet fully accessible. Currently the focus is on making Notebook 7 accessible.
```

Comment:
```suggestion
JupyterLab is not yet fully accessible. Currently, we are focused on making Notebook 7 accessible.
```

It might be good to clarify what you mean by "not yet fully accessible" and "accessible" — are we targeting a particular accessibility standard?